src/demo/Components/DocPage/index.tsx

```diff
@@ -61,10 +63,13 @@ const DocPageDemo = () => {
         console.log('Feedback:', data);
     }, []);
 
-    const onChangeBookmarkPage = useCallback((data: boolean) => {
-        setIsPinned(!data);
-        console.log(`This page pinned: ${data}`);
-    }, []);
+    const onChangeBookmarkPage = useCallback(
+        (data: boolean) => {
+            setIsPinned(data);
```

Comment:
`setIsPinned(!data)` -> `setIsPinned(data)`
is it OK?

Reply:
yes, it was mistake in demo in storybook